src/com/jalasoft/search_files/utils/Search_Validation_Test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+from src.com.jalasoft.search_files.utils.Search_Validation import SearchValidation
+
+
+class SearchValidationTest(unittest.TestCase):
+    def test_string1_exists_within_string2_method_return_true_if_string_1_exist_within_string2_with_case_sensitive_disable(self):
+        search_validation = SearchValidation()
+        string_1 = "LA mu"
+        string_2 = "HOLA mundo"
+        case_sensitive = False
+        value_boolean = SearchValidation.string1_exists_within_string2(
+            search_validation, string_1, string_2, case_sensitive)
+        self.assertTrue(value_boolean)
+
+    def test_string1_exists_within_string2_method_return_false_if_string_1_does_not_exist_within_string2_with_case_sensitive_disable(self):
+        search_validation = SearchValidation()
+        string_1 = "la MU"
+        string_2 = "HOLA mundo"
+        case_sensitive = False
+        value_boolean = SearchValidation.string1_exists_within_string2(
+            search_validation, string_1, string_2, case_sensitive)
+        self.assertFalse(value_boolean)
```

Comment:
In this case, title mentions that string_1_does_not_exist_within_string2 however if case_sensitive = False, and string_1 exist within string2, Why the assert is false ?